src/Meshes.jl

```diff
@@ -294,6 +294,8 @@ export
   OverlappingBoxes,
   FaceTouchingBoxes,
   CornerTouchingBoxes,
+  IntersectingSegmentTri,
```

Comment:
```suggestion
  IntersectingSegmentTri,
```

Please replace `Tri` by the explicit `Triangle` everywhere.